src/pages/create-timetable-page/CreateTimetablePage.tsx

```diff
@@ -1,9 +1,30 @@
 import { Button } from "@lib/gds-components";
+import {
+  DEFAULT_DEVELOPMENT_PLAN,
+  DEFAULT_TIMETABLE_EVENTS,
+} from "@lib/constants";
 import { Link } from "react-router-dom";
-import { PageRoute } from "../../routes/routes";
+import { PageRoute, Journeys } from "../../routes/routes";
+import { useFormContext } from "../../context/use-form-context";
 import styles from "./create-timetable.module.css";
+import { useEffect } from "react";
 
 export const CreateTimetablePage = () => {
+  const {
+    setTimetableEvents,
+    setDevelopmentPlan,
+    setLoadedTimetableEvents,
+    setLoadedDevelopmentPlan,
+    setUserFlow,
+  } = useFormContext();
+
+  useEffect(() => {
+    setTimetableEvents(DEFAULT_TIMETABLE_EVENTS);
+    setDevelopmentPlan(DEFAULT_DEVELOPMENT_PLAN);
+    setLoadedTimetableEvents(null);
+    setLoadedDevelopmentPlan(null);
+  });
```

Comment:
I think we might want empty deps on this so it doesn't happen every render?

Also something to think about in the future is whether we need to do the same thing on the other content pages